api/Window.json

```diff
@@ -4990,10 +4990,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": null
+              "version_added": false
```

Comment:
`ondevicemotion` (this) and `ondeviceorientation` (below) are set to true in https://github.com/mdn/browser-compat-data/pull/3728.

The event handler attributes are there, confirmed in Firefox Nightly. The event themselves might not fire on all platforms, but how to document that I don't know.

Reply:
A note should be included stating "Only in Firefox Nightly" if that's the case.  I was basing my code off of the tag marking the release of Firefox 65, so no nightly stuff.

Reply:
Firefox Nightly just happened to be the browser I had locally, it's in Firefox 65 too, and probably way back. In other words, something wrong with the methodology.

Reply:
Oh really?  If that's the case and it's in 65 as well, it's probably related to the above issue regarding partials...

I did some looking into the issue.  It turns out that the WebIDL parser library is more designed a little more towards code highlighting.  Partials, implements, and mixins don't get merged into their interfaces; they remain separate.

As such...looks like I'll have to redo this PR.  I was super skeptical about setting values to `false` initially, as I didn't trust my code all the way -- I should definitely trust my gut a lot more.  😛

Reply:
I can clearly see the event in the IDL files as an attribute to the interface, but even with the updates to check against all of the interface partials/etc., for some reason my script's not finding it.  I'll have to look into exactly why later on, for now, fixing this!